src/scss/_components.respondents.scss

```diff
@@ -0,0 +1,379 @@
+.cmp-respondents {
+  width: calc(100% - 2rem);
+  max-width: $max-width;
+  margin: 4rem auto 6rem;
+
+  @media (min-width: $bp-respond-medium-layout) {
+    margin: 6rem auto 8rem;
+  }
+
+  &__list {
+    @include delist;
+    margin-top: 4rem;
+
+    @supports (display: grid) {
+      display: grid;
+      grid-row-gap: 8rem;
+    }
+
+    @media (min-width: $bp-respond-start-grid) {
+      display: flex;
+      flex-wrap: wrap;
```

Comment:
In IE11, some of the items are really close together on small sizes (when they're still side by side, shown below). We can fix that by adding a `justify-content` here with `space-around` or `space-between`.

<img width="702" alt="Screen Shot 2019-07-02 at 2 42 32 PM" src="https://user-images.githubusercontent.com/23404383/60537957-a44e3000-9cd7-11e9-847e-fb26e7a5a951.png">